documentation/course/e2/README.md

```diff
@@ -852,53 +879,55 @@ TODO @piterek130: uzupełnić tabelę
   </tr>
   <tr>
     <th>Nazwa aplikacji</th>
-    <td>Aplikacja źródłowa (aplikacja, która inicjuje integrację na poziomie logicznym, czyli w większości przypadków aplikacja będąca przy końcu strzałki bez grotu)</td>
-    <td>Aplikacja docelowa (aplikacja przy końcu strzałki z grotem)</td>
+    <td>JakPrzyjade</td>
+    <td>Tpay</td>
   </tr>
   <tr>
     <th>Technika integracji</th>
-    <td>FTP/SSH/JDBC/ODBC/Oracle DBLink/SAP JCo/SAP RFC/ SOAP/HTTP / SOAP/HTTPS /Własny/...</td>
-    <td>FTP/SSH/JDBC/ODBC/Oracle DBLink/SAP JCo/SAP RFC/ SOAP/HTTP / SOAP/HTTPS /Własny/...</td>
+    <td>REST API / HTTPS</td>
+    <td>REST API / HTTPS</td>
   </tr>
   <tr>
     <th>Mechanizm autentykacji</th>
-    <td>Brak - niezalecane/HTTP Basic/WS-Security/Kerberos/Oracle Username and Password/...</td>
-    <td>Brak - niezalecane/HTTP Basic/WS-Security/Kerberos/Oracle Username and Password/...</td>
+    <td>OAuth 2.0</td>
+    <td>OAuth 2.0</td>
   </tr>
   <tr>
     <th>Kontrakt danych</th>
-    <td colspan="2">Lista wymienianych obiektów biznesowych (np. Partner Handlowy, Konto Umowy itp.)</td>
+    <td colspan="2">OAuth 2.0 Access Token, kwota, dane pasażera, opis transakcji, status transakcji</td>
```

Comment:
**thought:** Mentioning access token as a part of contract seems weird to me, but it is technically correct.